codesign/bin/main.dart

```diff
@@ -0,0 +1,122 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:codesign/codesign.dart';
+import 'package:platform/platform.dart';
+
+const String filepathFlag = 'filepath';
+const String helpFlag = 'help';
+
+const String kCommit = 'commit';
+const String kProduction = 'production';
+const String kCodesignCertName = 'codesign-cert-name';
+const String kCodesignPrimaryBundleId = 'codesign-primary-bundle-id';
+const String kCodesignUserName = 'codesign-username';
+const String kAppSpecificPassword = 'app-specific-password';
+const String kCodesignAppStoreId = 'codesign-appstore-id';
+const String kCodesignTeamId = 'codesign-team-id';
+const String kCodesignFilepath = 'filepath';
+
```

Comment:
Is there a specific reason we are prefacing constants with a k?  The dart style guide recommends against using prefixes.  [Effective Dart: Don't use prefix letters](https://dart.dev/guides/language/effective-dart/style#dont-use-prefix-letters).

Reply:
Cocoon generally follows the Flutter repo style guide.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k